### ISSUE TITLE
1.15.5 release had a build error on jitpack

### ISSUE THREAD

**Issue by kgabbott** [GetStream/stream-chat-android-client#81]
**Describe the bug**
The latest release cannot be installed since it there was a build error on jitpack.

See:
https://jitpack.io/#GetStream/stream-chat-android-client
https://jitpack.io/com/github/GetStream/stream-chat-android-client/1.15.5/build.log

**SDK version**
1.15.5

**Comment by tschellenbach** [comment]
Other repos use this version jacoco, @JcMinarro 

```
// build.gradle
classpath 'com.hiya:jacoco-android:0.2'
```

```
// livedata/build.gradle
apply plugin: "com.hiya.jacoco-android"
```

**Comment by tschellenbach** [comment]
We are looking into better build flows than jitpack, fixed now in 1.15.6 sorry about the inconvenience

Feel free to reach out if there's anything else we can help with

**Comment by JcMinarro** [comment]
@kgabbott the version 1.15.5 has been created properly.
I can see on the log link you provided it has a success build.
On the other hand, this build has been used some days ago to generate the [livedata](https://github.com/GetStream/stream-chat-android-livedata/releases/tag/0.7.6) build without any issue
